src/main/java/com/github/jasminb/jsonapi/ResourceConverter.java

```diff
@@ -507,9 +507,16 @@ private void handleRelationships(JsonNode source, Object object)
 							Collection elements = createCollectionInstance(relationshipField.getType());
 
 							for (JsonNode element : relationship.get(DATA)) {
-								Object relationshipObject = parseRelationship(element, type);
-								if (relationshipObject != null) {
-									elements.add(relationshipObject);
+								try {
+									Object relationshipObject = parseRelationship(element, type);
+									if (relationshipObject != null) {
+										elements.add(relationshipObject);
+									}
+								} catch (Exception ex) {
+									if(relationshipField.getType().isInterface() &&
```

Comment:
This needs to be changed in a way that proper exception is thrown from `parseRelationship` call which than should be caught and handled according to `DeserializationFeature`. Currently this will hide any issues in case allowing unknown types is enabled.

Formatting: `if (...`

Reply:
@Kevinrob | @jasminb 
I was going to make the requested changes, but after looking at this section closer, it isn't clear to me why we need this change.  I don't see any code in parseRelationship that would fail if the relationship was defined with an interface.  I'm assuming we would never annotate an interface with a `@Type` so the JsonNode should never resolve to an interface.

Can you let me know if I'm missing anything?  If there is a need for this, I want to add a similar catch around the `parseRelationship` call in the else statement below.

Reply:
@Kevinrob / @jasminb 
Hi guys, i've got all the requested changes done on this PR with the exception of this outstanding question. If you guys get a minute, can you add your feedback?